src/directory/getMetaStringObj.mjs

```diff
@@ -0,0 +1,48 @@
+import { promises as fs } from 'fs';
+import JSON5 from 'json5';
+
+export const invalidMetaError = (
```

Comment:
I pulled these error strings out originally because I thought it would be nice to check for the exact error thrown, but setting up the custom error types got a little more complicated than I expected 🙃 The rest of the file is a copy/paste from the original.